benches/bench.rs

```diff
@@ -859,9 +858,9 @@ fn http_access_log_lines() -> impl Iterator<Item = String> {
                 rng.gen::<u8>(), rng.gen::<u8>(), rng.gen::<u8>(), rng.gen::<u8>(), // IP
                 year.sample(&mut rng), mday.sample(&mut rng), // date
                 hour.sample(&mut rng), minsec.sample(&mut rng), minsec.sample(&mut rng), // time
-                rng.sample_iter(&Alphanumeric).take(url_size).collect::<String>(), // URL
+                rng.clone().sample_iter(&Alphanumeric).take(url_size).collect::<String>(), // URL
```

Comment:
Instead of `.clone` I think we want to use `(&mut rng).sample_iter(...)`

Reply:
Thanks. I definitely skip `(&mut self)` in docs. Did not thought about seed on clone, sorry.

Reply:
No worries, just figured I'd mention it for the future :smile: